docs/UserGuide.md

```diff
@@ -384,56 +442,133 @@ Format: `c-clear`
 
 #### 3.10 Archiving an employee : `c-archive`
 
-Archives the specified employee's contact detail from the tCheck.
+Archives the specified employee from tCheck.
 
 Format: `c-archive INDEX`
 
-* Archives the employee at the specified `INDEX`.
-* The index refers to the index number shown in the displayed `employees' contact details` panel.
-* The index **must be a positive integer** 1, 2, 3, ...​
+* Archives the employee at the specified `INDEX` in the _Employee Directory_ pane.<br>
+* The `INDEX` refers to the index number shown in the current _Employee Directory_ pane.<br>
+* The `INDEX` **must be a positive integer** (e.g. 1, 2, 3, ...​).<br>
+* The `INDEX` **should not be larger than** the total number of employees listed in the current 
+_Employee Directory_ pane. (e.g. There are 2 employees listed in the current _Employee Directory_ pane. The `INDEX`
+should be less or equal to 2.)<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+**:information_source: Notes:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of the name), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive INDEX` can be used to archive a specified employee identified
+  by `INDEX`.<br>
```

Comment:
Should this be placed in between the div line  to show the blue box?

Reply:
Thanks for pointing out! Will edit the div lines. Thanks